src/shared/chromium.ws.ts

```diff
@@ -12,6 +12,7 @@ import {
 import { Duplex } from 'stream';
 
 export interface QuerySchema extends SystemQueryParameters {
+  ignoreHTTPSErrors?: boolean;
```

Comment:
I believe this is part of the launch opts, no? Shim _should_ move it into the right place if so and we shouldn't need this here?

Reply:
It's not part of [Puppeteer's launch options](https://pptr.dev/browsers-api/browsers.launchoptions/#properties) :/. Doing requests using it throws an error:

```
Validating route query-params with QUERY schema
WebSocket query-params contain errors sending 400:"ignoreHTTPSErrors" is not allowed
```

Reply:
```import puppeteer from 'puppeteer-core';

const launch = {
  headless: 'boohoo',
  ignoreHTTPSErrors: true,
}

const job = async () => {
  try {
    const browser = await puppeteer.connect({
      browserWSEndpoint: `ws://localhost:3000/chromium?launch=${JSON.stringify(launch)}`,
    });

    console.log(await browser.version());
    const page = await browser.newPage();
    await page.goto('https://example.com');
    await page.close();
    await browser.disconnect();
  } catch (e) {
    console.log('Error', e);
  }
};

Array.from({ length: 1 }).map(job);```

Reply:
Just to confirm after digging into their code: they don't have it as an explicit launchOption, however, once they launch and connect then they pass in this value when they connect. So it is part of their launchOptions but usage of that value is deferred until they have connected.